### PR TITLE
Added a bunch of comments and TODOs!!!

### PR DESCRIPTION
**Pull request by GriffGreen**

### CODE REVIEW

File: contracts/LiquidPledgingBase.sol [hunk 1]
```diff
@@ -38,10 +40,10 @@ contract LiquidPledgingBase is Owned {
     uint constant MAX_INTERPROJECT_LEVEL = 20;
 
     enum PledgeAdminType { Giver, Delegate, Project }
-    enum PaymentState { Pledged, Paying, Paid }
+    enum PaymentState { Pledged, Paying, Paid } //TODO Very confusing with PaymentStatus rename to PledgeState??????????????
```
Comment (perissology) [inline contracts/LiquidPledgingBase.sol]: I agree

File: contracts/LiquidPledgingBase.sol [hunk 2]
```diff
@@ -64,7 +66,7 @@ contract LiquidPledgingBase is Owned {
         uint64 intendedProject; // Used when delegates are sending to projects
         uint64 commitTime;  // When the intendedProject will become the owner  
         uint64 oldPledge; // Points to the id that this Pledge was derived from
-        PaymentState paymentState; //  Pledged, Paying, Paid 
+        PaymentState paymentState; //  Pledged, Paying, Paid //TODO Very confusing with PaymentStatus rename to pledgeState??????????????
```
Comment (perissology) [inline contracts/LiquidPledgingBase.sol]: I agree

Comment (GriffGreen) [reply contracts/LiquidPledgingBase.sol]: Sweet, I don't know if it needs to delay the launch but i would like to leave the comments in and at least make it part of the next develop branch

File: contracts/LiquidPledgingBase.sol (hunk 1 above)
Comment (GriffGreen) [reply contracts/LiquidPledgingBase.sol]: Sweet, I don't know if it needs to delay the launch but i would like to leave the comments in and at least make it part of the next develop branch

**Pull request merged**